### DISCUSSION TITLE
测试

### DISCUSSION THREAD

**Discussion by AFUNGH** [share121/inter-knot#4875] [灌水]
![S40913-15232460_com taobao idlefish(1)(1)](https://github.com/user-attachments/assets/1a7c99f5-cf2b-4763-992c-ed3eb96eaaac)

**Comment by AFUNGH** [comment]
名字要怎么改？

**Reply by HLXS-HLXS** [reply]
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username#changing-your-username
